docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Contribution Guidelines
+
+As all source{d} projects, this project follows the
+[source{d} Contributing Guidelines](https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md).
+
+
+# Additional Contribution Guidelines
+
+In addition to the [source{d} Contributing Guidelines](https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md),
+this project follows the following guidelines.
+
+
+## Generated Code
+
+Before submitting a pull request make sure all the generated code changes are also committed.
+
+
+### kallax
+
+To generate go code from [kallax](https://github.com/src-d/go-kallax) models, run:
+
+```bash
+$ go generate ./...
+```
+
+To update [go-bindata](https://github.com/jteeuwen/go-bindata) with the new migration files:
+
+```bash
+$ kallax migrate --input ./store/models/ --out ./store/migrations --name <name>
+$ make dependencies
```

Comment:
wrong order of commands. Without `make dependencies` there is no kallax in $GOPATH/bin.